app/assets/stylesheets/steps.scss

```diff
@@ -20,6 +24,16 @@ pre.markdown-example {
   white-space: pre-wrap;
 }
 
+.nav.step-by-step-tabs {
+  margin-bottom: 2em;
+
+  @media screen and (min-width: 769px) and (max-width: 1000px) {
+    a {
```

Comment:
I think it would be useful if you got into the habit of only styling with classes, not using element selectors like this (as this is the GOV.UK way). I know it's more effort, because you have to apply a class to all the extra elements, but it's worth it in the long term for maintainability.